testing.md

```diff
@@ -0,0 +1,37 @@
+# Manual Testing (for Native Query)
+
+At this time, Native Query must be tested manually by reviewers.
+
+The following query is sufficient to test Native Query as it exercises mutliple areas of translation.
+
+1. Ensure you have a local ADF running.
+- Navigate to the `mongo-odbc-driver` repository locally.
+- Run the ADF script via `./resources/run_adf start`
+- After it sets up and returns back to the shell prompt, load local data via `cargo run --bin data_loader`
+- Navigate back to the `mongo-powerbi-connector` repository.
+
+2. Build the connector. The easiest way to do this is in VSCode, using the Power Query SDk.
+- Navigate to the `connector` directory.
+- `code .` to open the project in VSCode.
+- Open `MongoDBAtlasODC.pq`
+- Right click, and choose *Evaluate current power query file*
```

Comment:
```suggestion
- On the editor window, right click on  `MongoDBAtlasODC.pq`, and choose *Evaluate current power query file*
```
At least with my VSCode I have to click on the actual editor window, this might help if someone tries clicking on the filename

Reply:
good call